docs/source/notebooks/subset.ipynb

```diff
@@ -65,7 +65,7 @@
    "outputs": [],
    "source": [
     "pavics_url = 'https://pavics.ouranos.ca/twitcher/ows/proxy/finch/wps'\n",
-    "url = \"http://localhost:5000\" # os.environ.get('WPS_URL', pavics_url)\n",
+    "url = os.environ.get('WPS_URL', pavics_url)\n",
```

Comment:
FYI @aulemahal next time to avoid hardcoding like this and committing it by accident, you can simply start the notebook server with the var `WPS_URL` set and then the notebook will behave as if you have hardcoded a different url.

Something like this to start the notebook server:
`WPS_URL="http://localhost:5000" jupyter notebook`